tests/Resources/Page/OCPDashboard/Monitoring/Metrics.robot

```diff
@@ -23,11 +22,20 @@ Verify Query Results Dont Contain Data
   Should Be True   not ${metrics_query_results_contain_data}
 
 Run Query
-  [Arguments]  ${query}
+  [Arguments]  ${query}  ${retry-attempts}=6
   Input Text   ${METRICS_QUERY_TEXTAREA}  ${query}
   Press Keys   ${METRICS_QUERY_TEXTAREA}    ENTER
   Wait Until Page Does Not Contain    No query entered  timeout=20
 
+  ${metrics_query_results_contain_data} =  Run Keyword and Return Status   Verify Query Results Contain Data
+  IF  not ${metrics_query_results_contain_data}
+      IF    ${retry-attempts} > 0
+        ${retry-attempts}=   Evaluate    ${retry-attempts} - 1
+        Sleep  60  reason=Waiting for results to be ready
```

Comment:
do we need 6 mins?

Reply:
I think prometheus updates the info every 5 mins

Reply:
I've force pushed a new version with retry-attemps=10